src/bundle/Resources/config/ezplatform_default_settings.yml

```diff
@@ -24,3 +24,6 @@ parameters:
 
     # Universal Discovery Widget Module
     ezsettings.default.universal_discovery_widget_module.default_location_id: 1
+
+    # User identifier
+    ezsettings.default.user_identifier: ['user']
```

Comment:
I'm not convinced with the name. It should be more descriptive f.ex. `user_content_type_identifier`.